work/COVIDvu.ipynb

```diff
@@ -84,7 +84,7 @@
    "source": [
     "confirmedCases = pd.read_csv(CONFIRMED_FILE).groupby(['Country/Region']).sum()\n",
     "\n",
-    "confirmedCases"
+    "confirmedCases.head()"
```

Comment:
I thought about using `confirmedCases.tail()` but I liked how we saw the top and bottom in the original.  `head()` method it is!